tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -186,6 +204,11 @@ public:
    virtual RPage PopulatePage(ColumnHandle_t columnHandle, NTupleSize_t globalIndex) = 0;
    /// Another version of PopulatePage that allows to specify cluster-relative indexes
    virtual RPage PopulatePage(ColumnHandle_t columnHandle, const RClusterIndex &clusterIndex) = 0;
+
+   /// Populates all the pages of the given cluster id and columns; some of the columns might not contain
+   /// any pages, but the corresponding column ids must be registered in the cluster nevertheless.
+   /// LoadCluster in general runs in parallel to other methods of the page source.
```

Comment:
How does it run in parallel? Do you mean "can be run in parallel" or "will start threads that concurrently operate on the page source object"?

Reply:
Clarified the comment: the I/O thread of the cluster pool calls this function.  So implementers need to be aware that LoadCluster must be thread-safe.